### PR TITLE
STITCH-1369 - Implement AWS SES Service in Core, Android, and Server

### PR DESCRIPTION
**Pull request by edaniels**

### CODE REVIEW

**Comment by edaniels** [comment]
One sec forgot the SendResult

File: android/services/aws-ses/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<manifest package="com.mongodb.stitch.android.services.twilio">
```
Comment (jsflax) [inline android/services/aws-ses/src/main/AndroidManifest.xml]: [minor] this seems wrong

File: core/services/aws-ses/src/main/java/com/mongodb/stitch/core/services/aws/ses/CoreAwsSesServiceClient.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    this.service = service;
+  }
+
+  protected void sendEmailInternal(
+      final String toAddress,
+      final String fromAddress,
+      final String subject,
+      final String body
+  ) {
+    final Document args = new Document();
```
Comment (jsflax) [inline core/services/aws-ses/src/main/java/com/mongodb/stitch/core/services/aws/ses/CoreAwsSesServiceClient.java]: [minor] if I'm being a stickler, we can pull these keys out into consts, or, create a codec for this structure.

File: android/services/aws-ses/src/main/AndroidManifest.xml (hunk 1 above)
Comment (edaniels) [reply android/services/aws-ses/src/main/AndroidManifest.xml]: it is!

File: core/services/aws-ses/src/main/java/com/mongodb/stitch/core/services/aws/ses/CoreAwsSesServiceClient.java (hunk 2 above)
Comment (edaniels) [reply core/services/aws-ses/src/main/java/com/mongodb/stitch/core/services/aws/ses/CoreAwsSesServiceClient.java]: I think they don't really belong anywhere but here if they're used just once.

Comment (jsflax) [reply core/services/aws-ses/src/main/java/com/mongodb/stitch/core/services/aws/ses/CoreAwsSesServiceClient.java]: Alright, disregard then

**Pull request merged**